### PR TITLE
Issue #7200 fixed

### PR DESCRIPTION
**Pull request by vijay-webkul**
## Issue Reference
#7200

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@vijay-webkul 
**Not working**, **still** getting the **same** issue, please find the screenshot [Here](https://prnt.sc/ifTck57tCKPV)

**Comment by vijay-webkul** [comment]
I have checked not an issue, it is working properly See the screenshot [here](url)
Tax-Screenshot

File: packages/Webkul/Velocity/src/Resources/views/shop/checkout/onepage.blade.php [hunk 1]
```diff
@@ -400,6 +400,17 @@ class="theme-btn"
                                     if (address.last_name) {
                                         this.address.billing.last_name = address.last_name;
                                     }
+                                    
+                                    if(address.city){
```
Comment (Sanjaybhattwebkul) [inline packages/Webkul/Velocity/src/Resources/views/shop/checkout/onepage.blade.php:404]: add space after if and before {

File: packages/Webkul/Velocity/src/Resources/views/shop/checkout/onepage.blade.php [hunk 2]
```diff
@@ -400,6 +400,17 @@ class="theme-btn"
...
+                                        this.address.billing.city = address.city;
+                                    }
+                                    if(address.country){
+                                        this.address.billing.country = address.country;
+                                    }
+
+                                    if(address.state){
+                                        this.address.billing.state = address.state;
```
Comment (Sanjaybhattwebkul) [inline packages/Webkul/Velocity/src/Resources/views/shop/checkout/onepage.blade.php:412]: same here

File: packages/Webkul/Velocity/src/Resources/views/shop/checkout/onepage.blade.php [hunk 3]
```diff
@@ -400,6 +400,17 @@ class="theme-btn"
                                     if (address.last_name) {
                                         this.address.billing.last_name = address.last_name;
                                     }
+                                    
+                                    if(address.city){
+                                        this.address.billing.city = address.city;
+                                    }
+                                    if(address.country){
```
Comment (Sanjaybhattwebkul) [inline packages/Webkul/Velocity/src/Resources/views/shop/checkout/onepage.blade.php:407]: same here

**Review by Sanjaybhattwebkul — CHANGES_REQUESTED** [review]

**Comment by vijay-webkul** [comment]
all changes has done.

**Comment by Sanjaybhattwebkul** [comment]
@vijay-webkul 
**Not working**, the **checkout page** is getting **blank**.

**Comment by Sanjaybhattwebkul** [comment]
@vijay-webkul 
Any update?

**Review by Sanjaybhattwebkul — APPROVED** [review]

**Pull request closed without merging**